hotsos/core/plugins/storage/bcache.py

```diff
@@ -175,24 +175,76 @@ def is_bcache_device(self, dev):
         return False
 
 
-class BDevsConfig(BcacheBase):
+class BDevsInfo(BcacheBase):
 
-    def get(self, key):
-        """
-        This currently assumes there is only one cacheset on the host.
-        """
-        if self.cachesets and self.cachesets[0].bdevs:
-            return self.cachesets[0].bdevs[0].cfg.get(key)
+    @property
+    def sequential_cutoff(self):
```

Comment:
also since these values can be integers e.g. '100' or suffixed inetegers e.g. '1k' we need to take that into account when sorting. I have proposed a helper util to do this sorting if you want to take a look https://github.com/canonical/hotsos/pull/761